rules/how-to-read-source-code/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 title: Do you know how to read source code?
-uri: do-you-know-how-to-read-source-code
```

Comment:
Not sure if the following has been done, but just in-case,

1. Add a redirect to the old uri `do-you-know-how-to-read-source-code`
2. Update the categories with the new uri 
3. check that the folder is the same as the new uri

Reply:
> 1. Add a redirect to the old uri do-you-know-how-to-read-source-code
> 2. Update the categories with the new uri

All done

> 3. check that the folder is the same as the new uri

Done - it's the same

![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/37203901/dc09c4fa-1b15-4e97-99be-337df9208d9d)
**Figure: Folder name is consistent with the new URL**